modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsFieldsDisplayContext.java

```diff
@@ -129,6 +129,10 @@ public String[] getForbiddenLastCharacters() {
 		return forbiddenLastCharacters.toArray(new String[0]);
 	}
 
+	public Locale getLocale() {
+		return objectRequestHelper.getLocale();
+	}
+
```

Comment:
@marcelabc I think we don't need this here if we can use `locale` variable in the JSP.